components/logins/README.md

```diff
@@ -1,12 +1,15 @@
-# Logins
+# Logins Component
 
+![status-img](https://img.shields.io/static/v1?label=production&message=Lockwise,%20Firefox%20iOS&color=darkgreen)
+![status-img](https://img.shields.io/static/v1?label=not%20implemented&message=Firefox%20Preview,%20Desktop&color=darkred)
+
+## Implementation Overview
 Logins implements encrypted storage for login records on top of SQLcipher, with
-support for Sync (using the sync15 crate). It uses a slight modification on top
-of the database schema that firefox-ios used, however many of the queries and
-the way the sync is performed are different, as to allow syncs to complete with
-fewer database operations. See the header comment in `src/schema.rs` for an
-overview of the schema.
+support for Sync (using the [sync15](https://github.com/mozilla/application-services/tree/master/components/sync15) crate). It uses a slight modification on top
+of the database schema that [firefox-ios](https://github.com/mozilla-mobile/firefox-ios/blob/faa6a2839abf4da2c54ff1b3291174b50b31ab2c/Storage/SQL/SQLiteLogins.swift) used.  Notable difference include many of the queries and how sync is performed in order to allow syncs to complete with
```

Comment:
nit: differences.

Also, possibly worth noting that while the iOS schema used timestamps in microseconds, our timestamps are in milliseconds.